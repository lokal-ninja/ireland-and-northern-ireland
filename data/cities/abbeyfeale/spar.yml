id: 434862691
type: node
latitude: 52.3788
longitude: -9.30465
postcode: ""
city: Abbeyfeale
street: Abbeyfeale West
housenumber: ""
phone: ""
opening_hours: ""
website: ""
