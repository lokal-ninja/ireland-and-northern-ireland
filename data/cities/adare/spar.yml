id: 420075773
type: way
latitude: 52.54351
longitude: -8.85446
postcode: ""
city: Adare
street: ""
housenumber: 
phone: ""
opening_hours: ""
website: ""