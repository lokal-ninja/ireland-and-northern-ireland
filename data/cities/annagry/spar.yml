id: 747098304
type: node
latitude: 55.02065
longitude: -8.31089
postcode: ""
city: Annagry
street: ""
housenumber: ""
phone: ""
opening_hours: Mo-Sa 08:00-22:00; Su 09:00-21:00
website: ""
