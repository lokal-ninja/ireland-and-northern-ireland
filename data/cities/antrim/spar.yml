id: 220717740
type: way
latitude: 54.71365
longitude: -6.18884
postcode: "BT41 1JW"
city: Antrim
street: "Greystone Road"
housenumber: 
phone: ""
opening_hours: ""
website: ""