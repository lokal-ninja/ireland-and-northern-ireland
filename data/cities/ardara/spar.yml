id: 547719970
type: way
latitude: 54.762062400000005
longitude: -8.4150478
postcode: ""
city: Ardara
street: ""
housenumber: 
phone: ""
opening_hours: ""
website: ""
