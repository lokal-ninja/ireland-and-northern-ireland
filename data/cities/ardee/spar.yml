id: 4344893389
type: node
latitude: 53.85832
longitude: -6.54027
postcode: ""
city: Ardee
street: "Market Street"
housenumber: 
phone: "+353-41-6856349"
opening_hours: ""
website: ""