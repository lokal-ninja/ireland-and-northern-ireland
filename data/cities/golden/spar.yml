id: 518605749
type: way
latitude: 52.4987542
longitude: -7.980545500000001
postcode: ""
city: Golden
street: ""
housenumber: 
phone: ""
opening_hours: ""
website: ""
